sync/options.go

```diff
@@ -26,12 +26,16 @@ type Parameters struct {
 	// Keeping it private to disable serialization for it.
 	// default value is set to 0 so syncer will constantly request networking head.
 	blockTime time.Duration
+	// recencyThreshold describes the time period for which a header is
+	// considered "recent". The default is blockTime + 5 seconds.
+	recencyThreshold time.Duration
 }
 
 // DefaultParameters returns the default params to configure the syncer.
 func DefaultParameters() Parameters {
 	return Parameters{
-		TrustingPeriod: 168 * time.Hour,
+		TrustingPeriod:   168 * time.Hour,
+		recencyThreshold: time.Second * 20, // this default is based on a block time of 15 seconds
```

Comment:
This will fail our config roundtrip test. We can't set default values for private fields.

Reply:
It's fine to keep it empty though. But if really want a default, we can put in the getter method on Params struct.

Reply:
Also, this will allow to compute the default basing on blockTime